src/shared/components/EmissionAlertBadge.tsx

```diff
@@ -3,9 +3,25 @@ import { Translate } from 'react-i18nify';
 import { Alert } from 'reactstrap';
 
 export const EmissionAlertBadge = ({ emission }: any) => {
-    return (
-        <Alert color="warning">
-            <Translate value="emissions.yourEmissionsAreTooHigh" />
-        </Alert>
-    )
+    if(emission > 20){
+        return(
+             <Alert color="danger">
+                <Translate value="emissions.yourEmissionsAreTooHigh" />
+             </Alert>
+        )
+    }
+    else if(emission >= 10 && emission <= 20){
```

Comment:
@ewal2 well done, we can however simplify this logic a bit - we don't need to check the upper level of this condition here.